### PR TITLE
#2 - Implement react Form select on create/edit game screen

### PR DESCRIPTION
**Pull request by nazar**

### CODE REVIEW

File: client/src/components/Games/Create/CreateGame.js [hunk 1]
```diff
@@ -24,97 +35,128 @@ const CreateGame = class CreateGame extends Component {
...
-  }
+        })
+        .then(() => this.setState({ saved: true }));
+    }
+  };
+
+  valid = () => {
+    const errors = {};
+
+    _.isEmpty(this.formValue('title')) && (errors.title = 'Title is required');
```
Comment (mikelax) [inline client/src/components/Games/Create/CreateGame.js:132]: @nazar won’t these errors overwrite each other if there is more than one?  Shouldn’t the form input key be included in the errors prop?

File: client/.eslintrc [hunk 2]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "react-app"
+  "extends": "react-app",
+  "rules": {
+    "semi": 2     // come on! Gotta have ;
```
Comment (mikelax) [inline client/.eslintrc:4]: How was this not included in their ruleset?? :thinking

File: client/src/components/Games/Create/CreateGame.js [hunk 3]
```diff
@@ -24,97 +35,128 @@ const CreateGame = class CreateGame extends Component {
...
-              onChange={e => this.setState({ skillLevel: e.target.value })}
-            />
+              componentClass="select"
+              value={this.formValue('gameSettings.skillLevel')}
+              onChange={this.formInput('gameSettings.skillLevel')}>
+              <option value="1">Any/Newbie friendly</option>
+              <option value="2">I’ve rolled dice before</option>
+              <option value="3">Expert/role play master and rules bookworm</option>
+            </FormControl>
+
```
Comment (mikelax) [inline client/src/components/Games/Create/CreateGame.js:91]: Can you update this word to Frequency?

File: client/src/components/Games/Create/CreateGame.js [hunk 4]
```diff
@@ -24,97 +35,128 @@ const CreateGame = class CreateGame extends Component {
...
             <FormControl
-              type="text"
-              value={this.state.postingFrequency}
-              placeholder="Posting Frequence"
-              onChange={e => this.setState({ postingFrequency: e.target.value })}
-            />
+              componentClass="select"
+              value={this.formValue('gameSettings.postingFrequency')}
+              onChange={this.formInput('gameSettings.postingFrequency')}>
+              <option value="1">About 1 / day</option>
```
Comment (mikelax) [inline client/src/components/Games/Create/CreateGame.js:97]: @nazar what happens if we have to display the selected value elsewhere in app?  

Or on the search results page?  Should they (display values) be contained in a common location

**Comment by mikelax** [comment]
also forgot to ask ,  is the call to this.forceUpdate() an Apollo or react function?

Are you using that because you didn’t use setState function?

File: client/.eslintrc (hunk 2 above)
Comment (nazar) [reply client/.eslintrc:4]: I'm not surprised - this is what  happens when the java/c# class/deep nested inheritance trees crowd flock to javascript. First mixins are outlawed and everything turns into HoC of a HoC of a HoC (see a pattern?)

Joking! 😸

File: client/src/components/Games/Create/CreateGame.js (hunk 4 above)
Comment (nazar) [reply client/src/components/Games/Create/CreateGame.js:97]: That's a simple refactor to extract the key and descriptions to a function which can be reused in other places in the app.

File: client/src/components/Games/Create/CreateGame.js (hunk 1 above)
Comment (nazar) [reply client/src/components/Games/Create/CreateGame.js:132]: oops - typo

**Comment by nazar** [comment]
>is the call to this.forceUpdate() an Apollo or react function?

`ForceUpdate` is a react function - these should no longer be present in the PR

**Review by mikelax — APPROVED** [review]
🚀

**Pull request merged**